### ISSUE TITLE
Version 1.1.0 not working; show only "Loading..."

### ISSUE THREAD

**Issue by DengelFred** [joplin/plugin-kanban#51]
Hi,

any chance to download the version 1.0.7 anywhere? Sorry, I'm searching since hours and found nothing...
Btw. There is no Logfile available... therefore I cannot help to fix it...

Thanks a lot, DengelFred

**Comment by benlau** [comment]
hi @DengelFred ,

I can't reproduce the issue. What is your Joplin version? Could you try this sample kanban and see can it load?

```kanban
filters:
  tags: non-existed-tag
columns:
  - name: Backlog
    backlog: true
```

You may download the old version of plugin here:

https://registry.npmjs.org/joplin-plugin-kanban/-/joplin-plugin-kanban-1.0.7.tgz

Uncompress the file, inside the publish folder, you could find a jpl file for install locally.

**Comment by DengelFred** [comment]
Thanks for the link :-)
I tried your example. same behavior and no logfile.

Joplin 2.14.22 (prod, linux)
CodePerfect: 1.0.2
Favorites: 1.3.2
Inline TODO: 1.7.1
Joplin DDDot: 0.4.2
Kanban: 1.1.0
Markdown Table: Sortable: 1.2.2
MultiMarkdown Table Tools: 1.2.1
Note Tabs: 1.4.0
Persistent Editor Layout: 2.2.0
Quick HTML tags: 0.2.0
Search & Replace: 2.2.0

**Comment by DengelFred** [comment]
I can see the "Loading" for the 1.0.7 as well, but only for a few seconds.

**Comment by DengelFred** [comment]
unfortunately, I really need the "ignore tag" feature. Let me know if I can help.

**Comment by VladPetruM** [comment]
I have the same problem as well. Using joplin 3.0.14 for arm on mac os

**Comment by Computer-Ron** [comment]
I too have encountered this issue. This morning, after performing an upgrade to 3.0.14 with no change, chose "Reset application layout" under View which resolved it. I use multiple profiles (6 presently, segmented by different functions) and do customize my layout on all my desktops/laptops. It would appear that during usage (this has happened before) the layout is becoming corrupted (hence why it shows only "...Loading" with no way to close it short of disabling the plugin). Or, resetting my layout.

I am using plugin version 1.1.0, use Joplin on Android, Linux and Windows 10. The issue, when it has appeared, affects only a single machine (most recently my Ubuntu Linux desktop). I cannot say what preceded or triggered the behavior/event to happen. Hopefully this information proves useful.

**Comment by Computer-Ron** [comment]
Further analysis reveals I can reproduce the issue on two machines, both updated to 3.0.14 running plugin 1.1.0 by having a modified layout. After resetting the layout and reconfiguring it after closing and reopening the "Loading..." issue reappears.
![Joplin Loading-2024-08-06](https://github.com/user-attachments/assets/617b0163-7f25-4dbb-b7de-56d7db3da210)

**Comment by DengelFred** [comment]
Indeed, the reset and reconfiguration of the layout seems to solve the problem (Joplin 3.0.13)

**Comment by Computer-Ron** [comment]
This issue, I believe, is due to an incompatibility with the **Outline** plugin, presently v1.5.13. As a workaround I am using one specific notebook for/with the Kanban plugin where I have disabled the Outline plugin and subsequently don't appear to encounter the issue. In my other notebooks I have disabled the Kanban plugin in favor of the Outline plugin.

**Comment by Computer-Ron** [comment]
Additional information; I've enabled the kanban plugin on only one profile. Within the profile I've disabled the Outline plugin and there exists only one main notebook (named Change in this instance) and have crated multiple sub-notebooks (computer1, computer2 etc. for example) under it. The kanban "code" exists only within the master/top notebook (Change). When making an entry (to-do's in this case) in a sub-notebook and then leaving the sub-notebook selected, exiting and re-starting the Joplin app, the "Loading" issue reappears. However, if I first select the top notebook before exiting/closing the app (the Change notebook in this case) the "Loading..." issue does not occur.

I am able to "resolve" the "Loading..." display issue (temporarily, until what is described above occurs again) by selecting the top notebook and then navigating to Tools, Options, Plugins, Back.

### Joplin Instance Information:
Joplin 3.0.15 (prod, linux)

Client ID: REDACTED
Sync Version: 3
Profile Version: 47
Keychain Supported: No

Revision: 598677b

Backup: 1.4.1
Combine notes: 1.2.2
Conflict Resolution: 1.2.3
Csv Import: 1.0.1
Insert Date: 1.0.1
Kanban: 1.1.0
Markdown Table: Sortable: 1.2.2
Quick HTML tags: 0.2.0
Quick Links: 1.3.2
Quick Move: 1.1.0
Reset Checkboxes: 1.0.3
Search & Replace: 2.2.0
Table Formatter Plugin: 1.2.1
Templates: 2.4.0
Text Colorize: 1.2.5

### Plugin Note Code:
```
```kanban
columns:
  - name: Approved
    backlog: true
  - name: Work in progress
    tag: wip
  - name: Finished
    completed: true
```
```
